lib/ts/recipe/dashboard/api/getTenantLoginMethodsInfo.ts

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import { APIInterface, APIOptions } from "../types";
+import Multitenancy from "../../multitenancy";
+import { ProviderConfig } from "../../thirdparty/types";
+import PasswordlessRecipe from "../../passwordless/recipe";
+import ThirdPartyPasswordlessRecipe from "../../thirdpartypasswordless/recipe";
+import EmailPasswordRecipe from "../../emailpassword/recipe";
+import ThirdPartyEmailPasswordRecipe from "../../thirdpartyemailpassword/recipe";
+import ThirdParty from "../../thirdparty/recipe";
+import { TypeNormalisedInput } from "../../passwordless/types";
+
+type PasswordlessContactMethod = TypeNormalisedInput["contactMethod"];
+
+type TenantLoginMethodType = {
+    tenantId: string;
+    emailPassword: {
+        enabled: boolean;
+    };
+    passwordless: {
+        enabled: boolean;
+        contactMethod?: PasswordlessContactMethod;
+    };
+    thirdParty: {
+        enabled: boolean;
+        providers: ProviderConfig[];
+    };
+};
+
+export type Response = {
+    status: "OK";
+    tenants: TenantLoginMethodType[];
+};
+
+export default async function getTenantLoginMethodsInfo(
+    _: APIInterface,
+    __: string,
+    ___: APIOptions,
+    userContext: any
+): Promise<Response> {
+    const tenantsRes = await Multitenancy.listAllTenants(userContext);
```

Comment:
why are you calling list all tenants? There is a function to get the tenant info for just a specific tenant id. Why arent you using that?

Reply:
This API needs to be per tenant right?

Reply:
Addressed this issue on slack in our discussion.